packages/fx-core/resource/package.nls.json

```diff
@@ -401,6 +401,8 @@
   "core.common.ResolveServerUrlFailed": "Unable to resolve the server URL: please make sure that the environment variable %s is defined.",
   "core.common.OperationOnlyContainsOptionalParam": "Operation %s only contains optional parameters which are not supported",
   "core.common.ConvertSwaggerToOpenAPI": "The Swagger 2.0 file has been converted to OpenAPI 3.0.",
+  "core.common.SentApiRequest": "Sending API request: %s. Request body: %s",
```

Comment:
please check whether it is "Sending" or supposed to be "Sent"?

Reply:
Should be sending, and I have rename the key name. Thanks!